src/sass/partials/base/_base.scss

```diff
@@ -0,0 +1,312 @@
+
+._wrapper {
+  @include wrapper;
+}
+
+.base {
+  &--a {
+    border-bottom: 2px solid setting-get('primary color');
+    color: setting-get('primary color');
+    text-decoration: none;
+
+    &:hover {
+      border-color: setting-get('secondary color');
+      color: setting-get('secondary color');
+      transition: setting-get('fade');
+    }
+  }
+
+  // Major Third Type Scale
+
+  &--h1 {
+    font-size: 2.441em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h2 {
+    font-size: 1.953em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h3 {
+    font-size: 1.563em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h4 {
+    font-size: 1.25em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h5 {
+    font-size: 1em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--h6 {
+    font-size: .8em;
+    font-weight: setting-get('normal font weight');
+  }
+
+  &--p {
+    font-weight: setting-get('normal font weight');
+    @include breakpoint(setting-get('max paragraph length')) {
+      max-width: 40em;
+    }
+  }
+
+  &--blockquote {
+    border-left: .25em solid setting-get('primary accent color');
+    color: color('gray', 80);
+    font-style: italic;
+    margin-left: 2em;
+    margin-right: 1em;
+    padding: 0 1.5em;
+  }
+
+  &--dt {
+    color: setting-get('secondary color');
+    font-weight: setting-get('bold font weight');
+  }
+
+  &--dd {
+    @include breakpoint(setting-get('max paragraph length')) {
+      max-width: 40em;
+    }
+  }
+
+  &--li {
+    margin-top: 0;
+
+    &::before {
+      content: '•';
+      margin-left: -1em;
```

Comment:
* Eek. Let's avoid negative margins as much as possible.
* Why are you doing this for list items instead of using the standard bullets?